app/models/user.rb

```diff
@@ -73,7 +81,19 @@ def send_password_reset_email
   end
 
   def feed
-    microposts.by_created_at
+    Micropost.where("user_id IN (?) OR user_id = ?", following_ids, id).recent_posts
```

Comment:
`.where("user_id IN (?) OR user_id = ?", following_ids, id)` scope nhé

Reply:
Dạ đoạn này em dùng lại cái `.recent_posts` để scope theo thời gian rồi